### PR TITLE
Removed connection config classes

### PR DESCRIPTION
**Pull request by louis89**

### CODE REVIEW

File: californium-core/src/main/java/org/eclipse/californium/core/network/config/DefaultTCPConnectionConfig.java [hunk 1]
```diff
@@ -1,26 +0,0 @@
-package org.eclipse.californium.core.network.config;
-
-import org.eclipse.californium.elements.config.TCPConnectionConfig;
-
-public class DefaultTCPConnectionConfig extends TCPConnectionConfig{
```
Comment (joshlarson) [inline californium-core/src/main/java/org/eclipse/californium/core/network/config/DefaultTCPConnectionConfig.java]: Did you want to delete the TCPConnectionConfig class too?

Comment (louis89) [inline californium-core/src/main/java/org/eclipse/californium/core/network/config/DefaultTCPConnectionConfig.java]: > Did you want to delete the TCPConnectionConfig class too?

Yes. And I did in the element.connector PR.

Comment (joshlarson) [inline californium-core/src/main/java/org/eclipse/californium/core/network/config/DefaultTCPConnectionConfig.java]: Nevermind...

**Comment by joshlarson** [comment]
Approve

**Comment by jschloman** [comment]
Approve

**Pull request merged**